### PR TITLE
Google Drive Link

### LINKED ISSUE #21: Add Google Drive link on Resource page
_Requested by Comprehensive Exam Guide team_

On the resource page, the CEG team requested that Tech add a link directing to the Google Drive, preferably on the tabs.

### PR DESCRIPTION
**Pull request by AARandom**
Added a google drive link on the resource page as per Comprehensive Exam Guide team's request

Fixes #21 

## Type of change

- New feature (non-breaking change which adds functionality)

*Please delete options that are not relevant.*

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the website locally with my changes, and I am sure that it works well on different screen sizes

### CODE REVIEW

**Comment by AARandom** [comment]
// copy and paste from the issues comment //

As per their request, I have made a preview:
image

CEG chose the tab link and I agree as the second option looks very out of place

However, the problem arose on smaller screens, when the tabs are arranged weirdly.

image

image

Anyone has any suggestions?

**Comment by jaredliw** [comment]
I suggest doing something like:
image

with text "View on Google Drive"

File: layouts/page/resources.html [hunk 1]
```diff
@@ -20,16 +21,27 @@ <h2 class="text-center text-primary fw-lighter">
                 </li>
                 {{ $first = false }}
             {{ end }}
+
+            <li class="nav-item ms-auto">
+                <a class="nav-link text-decoration-underline" href="https://drive.google.com/drive/folders/1rjg6h0bArc4QbKTAdAV6O8mYe5dVYlil?usp=drive_link" target="_blank">Drive Link</a>
```
Comment (jaredliw) [inline layouts/page/resources.html:26]: May you put this link into `config.toml`?

Comment (AARandom) [reply layouts/page/resources.html:26]: hey jared, thanks for the suggestion! Lemme try it out on my end

**Comment by AARandom** [comment]
Its looks better now
Thanks jared

image

**Review by lynngoh7 — APPROVED** [review]
Hi, looks good to me!

**Pull request merged**